tests/test_urlparse.py

```diff
@@ -989,6 +1007,7 @@ def test_all(self):
         self.assertCountEqual(urlparse4.__all__, expected)
 
 
+@pytest.mark.skip(reason="We dont need this test")
```

Comment:
If we're not going to need this in the future, it's fine to remove it completely